src/components/Reactions/QuickEmojiReactions/BaseQuickEmojiReactions.js

```diff
@@ -32,9 +32,20 @@ const baseQuickEmojiReactionsPropTypes = {
     onPressOpenPicker: PropTypes.func,
 };
 
+const baseQuickEmojiReactionsDefaultProps = {
+    onEmojiSelected: () => {},
+    onWillShowPicker: () => {},
+    onPressOpenPicker: () => {},
+};
+
 const propTypes = {
     ...baseQuickEmojiReactionsPropTypes,
-    preferredSkinTone: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
+    preferredSkinTone: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+};
+
+const defaultProps = {
+    ...baseQuickEmojiReactionsDefaultProps,
+    preferredSkinTone: undefined,
```

Comment:
same thing here, I think a const is the way to go since we're doing this in multiple places